tests/utils.py

```diff
@@ -8,6 +8,12 @@
     if env_spec.entry_point.startswith("minigrid.envs")
 ]
 
+minigrid_testing_env_specs = [
+    env_spec
+    for env_spec in all_testing_env_specs
+    if not env_spec.entry_point.startswith("minigrid.envs.babyai")
```

Comment:
Why are we not testing babyai? 

Currently the wrapper tests against all of the minigrid. As the number of environments has just doubled we should probably remove this but should mean that we can test everything within a reasonable amount of time

Reply:
Ohh I think I understand now.